treon/treon.py

```diff
@@ -106,6 +109,13 @@ def print_test_collection(notebooks):
     LOG.debug(message)
 
 
+def filter_results(results, args):
+    for exclude_str in args['--exclude']:
+        exclude_abs = os.path.abspath(exclude_str)
```

Comment:
When I try os.path.abspath on python console it seems to append everything to current path.
```
>>> os.path.abspath('~/workspace')
'/Users/amitrathi/workspace/treon_forks/midokura-silvia/treon/~/workspace'
```

We might need this - 

```
>>> os.path.expanduser('~/workspace')
'/Users/amitrathi/workspace'
```

https://docs.python.org/3/library/os.path.html#os.path.expanduser

Bonus point if you could add a test for tilda (~) expansion :)

Reply:
yes, I noticed this problem as well